src/tests/utests/xi_utest_mqtt_logic_layer_subscribe.c

```diff
@@ -134,14 +134,15 @@ XI_TT_TESTCASE_WITH_SETUP(
         // set the task data
         XI_ALLOC_AT( xi_mqtt_logic_task_t, task, local_state );
 
-        task->cs = 103; // this is very hakish since it depends on the code
+        task->cs = 109; // this is very hakish since it depends on the code
         // so most probably this test will fail everytime we change anything in
```

Comment:
This is not elegant solution as it's adds manual work and only makes writing tests easier, not writing code.